ChatApp/Controlers/ChatViewController.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  ViewController.swift
+//  ChatApp
+//
+//  Created by bakhva  on 19.04.23.
+//
+
+import UIKit
+
+final class ChatViewController: UIViewController {
+    
+    let viewModel = ChatViewModel()
+    
+    // MARK: Outlets
+    lazy var dividerView: UIView = {
```

Comment:
I guess there is no need to use lazy here

```suggestion
    private let dividerView: UIView = {
```